packages/app-project/src/screens/ProjectHomePage/ProjectHomePage.js

```diff
@@ -70,10 +71,16 @@ function ProjectHomePage ({
       <Media at='default'>
         <header>
           <ZooHeaderWrapper isAdmin={adminMode} />
-          <ProjectHeader adminMode={adminMode} />
+          <ProjectHeader
+            adminMode={adminMode}
+            organization={organization}
```

Comment:
Out of interest, why isn't the organisation stored in the global store, like the project?